### PR TITLE
Chore/runtime uprgade scripts

### PR DESCRIPTION
**Pull request by saraswatpuneet**
# Goal
The goal of this PR is  to add mechanism for runtime upgrade for frequency for #63 

image

Closes: #63 

# Checklist
- [x] Scripts updated
- [x] make file update
- [x] Test for local upgrade passed
- [x] Readme updated

### CODE REVIEW

**Comment by shannonwells** [comment]
How is the js/onboard stuff intended to be used?  Also this is _just_ for upgrading locally right?

**Comment by saraswatpuneet** [comment]
@shannonwells  yes, and this will lay foundation for testnet too but there we will need scheduler and possibly all that will be manual for sometime

**Review by wilwade — APPROVED** [review]
Worked for me locally!

File: README.md [hunk 1]
```diff
@@ -155,6 +155,19 @@ in `rustfmt.toml`.
    TARGET=build-runtime RUST_TOOLCHAIN=nightly ./ci/build.sh
    ```
 
+# Runtime Upgrade
```
Comment (enddynayn) [inline README.md:158]: maybe label this Local Runtime Upgrade

**Review by enddynayn — APPROVED** [review]
Fabulous!

**Comment by enddynayn** [comment]
Looks great. I would have expected to see something like `api.tx.sytem.setCode()` called somewhere. Curious why isn't it called?

File: README.md [hunk 2]
```diff
@@ -155,6 +155,19 @@ in `rustfmt.toml`.
...
+
+1. Runtime upgrade enables upgrading running chain to a new runtime.
+2. To upgrade the runtime, current scripts does following step process:
+   1. Build new runtime and generate the compressed wasm
+   2. Call ```authorizeUpgrade``` extrinsic from parachain system to initate the upgrade.
+   3. Call ```enactAuthorizedUpgrade``` extrinsic from parachain system to enact the upgrade.
+   4. For testnet and mainnet, the upgrade is done slightly differently using ```scheduler``` and enactment is scheduled for a specific block number in the future.
```
Comment (shannonwells) [inline README.md:165]: 👍

**Review by shannonwells — APPROVED** [review]
looks good to me, thanks for adding that scheduler bit

**Pull request merged**